doc/releasenotes/10_0_4_release_notes.md

```diff
@@ -15,6 +15,8 @@ This patch is providing an update regarding the Apache Log4j security vulnerabil
 ------------
 ## Changelog
 
+- `vtctl ApplySchema -uuid_list='...'` now rejects a migration if an existing migration has the same UUID but with different `migration_context`.
+
```

Comment:
Wrong file? This is the 10.0.4 release notes.

Reply:
wrong file indeed